docs/content/quickstart/_index.md

```diff
@@ -11,200 +11,120 @@ Can't wait to try out the PostgreSQL Operator? Let us show you the quickest poss
-
-Lines that start with a `#` are commented out. To activate that configuration setting, you will have to delete the `#`.
+If not, please read onward: you can still get up and running fairly quickly with just a little bit of configuration.
 
-Set up your `inventory` file based on one of the environments that you are deploying to:
+## Step 1: Configuration
 
-#### Kubernetes
+### Get the PostgreSQL Operator Installer Manifest
 
-You will have to uncomment and set the `kubernetes_context` variable. This can be determined based on the output of the `kubectl config current-context` e.g.:
+You will need to download the PostgreSQL Operator Installer manifest to your environment, which you can do with the following command:
 
-```shell
-kubectl config current-context
-kubernetes-admin@kubernetes
 ```
-
-Note that the output will vary based on the Kubernetes cluster you are using.
-
-Using the above example, set the value of `kubernetes_context` to the output of the `kubectl config current-context` command, e.g.
-
-```python
-kubernetes_context="kubernetes-admin@kubernetes"
+curl https://raw.githubusercontent.com/CrunchyData/postgres-operator/master/installers/kubectl/postgres-operator.yml > postgres-operator.yml
 ```
 
-Find the location of the `pgo_admin_password` configuration variable. Set this to a password of your choosing, e.g.
+If you wish to download a specific version of the installer, you can substitute `master` with the version of the tag, i.e.
 
-```python
-pgo_admin_password="hippo-elephant"
 ```
-
-Finally, you will need to set the storage default storage classes that you would like the Operator to use. For example, if your Kubernetes environment is using NFS storage, you would set this variables to the following:
-
-```python
-backrest_storage='nfsstorage'
-backup_storage='nfsstorage'
-primary_storage='nfsstorage'
-replica_storage='nfsstorage'
+curl https://raw.githubusercontent.com/CrunchyData/postgres-operator/v4.3.0/installers/kubectl/postgres-operator.yml > postgres-operator.yml
 ```
 
-For a full list of available storage types that can be used with this installation method, see: $URL
-
-#### OpenShift
-
-For an OpenShfit deployment, you will at a minimum have to to uncomment and set the `openshift_host` variable. This is the location of where your OpenShift environment is, and can be obtained from your administrator. For example:
-
-```python
-openshift_host="https://openshift.example.com:6443"
-```
+### Configure the PostgreSQL Operator Installer
 
-Based on how your OpenShift environment is configured, you may need to set the following variables:
+There are many [configuration parameters](<{{< relref "/installation/configuration.md">}}>) to help you fine tune your installation, but there are a few that you may want to change to get the PostgreSQL Operator to run in your environment. Open up the `postgres-operator.yml` file and edit a few variables.
```

Comment:
I think this link should be `({{< relref "/installation/configuration.md">}})`

Reply:
Yup, thanks!